src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -946,7 +947,7 @@ private void testValidationReload() throws Exception {
                 TimeHelper.parseLocalDateTime("01/04/2035", "10", "00"),
                 TimeHelper.parseLocalDateTime("30/04/2035", "22", "00"),
                 null, null,
-                newSession.getInstructions(), newSession.getGracePeriod(), -2.0);
+                newSession.getInstructions(), newSession.getGracePeriod(), TimeHelper.convertToZoneId(-2));
```

Comment:
Just use something like `ZoneId.of("-02:00")`? `convertToZoneId` should be removed eventually.